doc/ref_impl/cntt-ri/chapters/chapter03.md

```diff
@@ -4,16 +4,123 @@
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [3.1 Introduction](#2.1)
-* [3.2 Heading](#2.2)
-* [3.3 Heading](#2.3)
+* [3.1 Introduction](#3.1)
+* [3.2 VNF profile](#3.2)
+* [3.3 NFVI SW profile](#3.3)
+* [3.4 NFVI HW profile](#3.4)
 
 <a name="3.1"></a>
 ## 3.1 Introduction
 
+Three types of profiles are mentioned in CNTT Reference Model: VNF profile is used to describe every workload running on top of NFVI. NFVI SW profile is used to describe the list of features provided by the hypervisor and host OS. Finally the NFVI HW profile is used to depict the details characteristics of every actual servers on the bottom.
+
+The CNTT Reference Model will be referenced as **RM1** to avoid long and duplicated reference titles.
+
 <a name="3.2"></a>
-## 3.2 Heading
+## 3.2 VNF profile
+
+Any virtual network functions and/or cloud-native network functions must choose to run on one of the pre-defined of entries in NFVI Infrastructure Profiles Catalogue. As states in [RM1: 4 Infrastructure Capabilities, Measurements and Catalogue: 4.2 Catalogue](../../../ref_model/chapters/chapter04.html#4.2), the entry uses the following naming convention.
```

Comment:
You need to use links to `.md` files instead of `.html`. Currently these will be broken for the user who is reading them from github repo instead of `github.io`. Same change for other places.

Reply:
Thanks, I will fix that.